### DISCUSSION TITLE
Error 422 Unprocessable Entity when settting up email IMAP

### DISCUSSION THREAD

**Discussion by Jatin-MYST** [chatwoot/chatwoot#5418] [Self hosted]
![image](https://user-images.githubusercontent.com/72418976/189560764-45122d08-cf49-45e4-a17c-6c0d01e28b18.png)
![image](https://user-images.githubusercontent.com/72418976/189560840-8b2c36bf-c6d6-4407-b76e-d0e44ebb1389.png)

How can I go about fixing this error?

**Comment by vikneo2017** [comment]
I have a same error. Please, @Jatin-MYST do you have solution for this problem?

**Comment by Jatin-MYST** [comment]
@vikneo2017 I recommend that you try a manual IMAP Login test via the terminal on the same machine that you have chatwoot installed. This way is much more helpful in finding the problem. I did the same and found out that the problem was on Microsofts end. (I was IMAPing to an outlook account)

Here are some links to get you started:
https://www.funoracleapps.com/2022/01/how-to-access-imap-server-from-command.html
https://stackoverflow.com/questions/14959461/how-to-talk-to-imap-server-in-shell-via-openssl 
https://www.atmail.com/blog/imap-101-manual-imap-sessions/ 

Good luck!

PS. If you run into the same problem with manual IMAP not replying back. it fixed it with adding '-crlf'.

**Comment by bfontaine** [comment]
Well I have the same problem with SMTP. The API response is a 422 (Unprocessable Content).
